src/main/java/com/alex/eshop/eshop/RestController/CategoryController.java

```diff
@@ -0,0 +1,28 @@
+package com.alex.eshop.eshop.RestController;
+
+import com.alex.eshop.eshop.Entity.Category;
+import com.alex.eshop.eshop.Service.CategoryService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api")
+public class CategoryController {
+    private final  CategoryService categoryService;
```

Comment:
Please, use **Ctrl + Alt + L** for every file to apply standard formatting for the code. Each function must be separated with empty row, so it will be easier to read functions.